### PR TITLE
Pricing Screen enhancements

### PR DESCRIPTION
**Pull request by sowmiyasubramaniyan**
Run the following command: 
php artisan migrate
php artisan db:seed --class="Database\Seeders\v3_0_3\DatabaseSeeder"

### CODE REVIEW

**Review by ajaz-ul-haq — DISMISSED** [review]

File: resources/lang/en/message.php [hunk 1]
```diff
@@ -0,0 +1,698 @@
+<?php
```
Comment (sandesh556) [inline resources/lang/en/message.php:1]: remove this file if lang is already present outside

**Review by sandesh556 — CHANGES_REQUESTED** [review]

File: resources/lang/en/message.php (hunk 1 above)
Comment (sowmiyasubramaniyan) [reply resources/lang/en/message.php:1]: It has been removed in the release build, I will rebase it once the release is done.

**Review by sandesh556 — APPROVED** [review]

**Comment by bhanu2217** [comment]
**Feedback:**

- Contact sales text discuss with Dhanya not looking good.
- Store group, give an option to show monthly yearly plans or drop down like it currently shows.

**Comment by bhanu2217** [comment]
On first time page load, by default yearly price should be selected.

**Review by SarfrazS — CHANGES_REQUESTED** [review]
When we open group getting below error. I cant continue testing. Please fix it and let me know
image

**Review by SarfrazS — CHANGES_REQUESTED** [review]
All functionalities are not implemented yet i showed to Sandesh and Sowmiya. Sowmiya will work on it once done then will start testing.

Issue1. Offer percentage not working.
image
After save 
image

Issue2. Price is not calculating properly on the store page.
image

Issue3. Contactus sales button is not there because unable to create product getting below error.
image
image

Issue4. Order button alignment is not similar to other products.
image

Issue5. On the group, we have a checkbox with monthly/yearly but on the client, panel group shows yearly/monthly.
image

image

Issue6. Regular price should be split based on the plan period as of now showing the regular price per month.

Issue7. In ServiceDesk Perpetual group we didn't check monthly/yearly but it is reflected here also and the lang file missing.
image
image




Please fix the 3rd issue on priority then only i can complete testing first round.

**Comment by sowmiyasubramaniyan** [comment]
@SarfrazS 3rd and 9th issues are fixed.

**Pull request closed without merging**